Mathlib/GroupTheory/Finite.lean

```diff
@@ -0,0 +1,20 @@
+/-
+Copyright (c) 2025 Junyan Xu. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Junyan Xu
+-/
+module
+
+public import Mathlib.Algebra.Regular.Basic
+public import Mathlib.Data.Fintype.Card
+
+/-!
+# Finite monoids are Dedekind-finite
```

Comment:
I think this file has the wrong name. `Mathlib/GroupTheory/Finite.lean` sounds like it should contain anything about finite groups but this.